404.html

```diff
@@ -9,21 +9,7 @@ <h1 class="text-lowercase page-header">Oops</h1>
     <h2 class="tagline push-bottom">We're sorry. This page doesn't seem to exist.</h2>
   </div>
   <div class="col-sm-6">
-    <p>Please use the navigation above or search here.</p>
-    <form role="form" action="{{ site.search_domain }}/search">
-      <div class="form-group">
-        <div class="input-group input-group-lg">
-          <input class="form-control input-lg" placeholder="Search..." type="search" name="q" data-automation-id="404-search-field">
-          <span class="input-group-btn">
-            <button type="submit" class="btn btn-blue">
-              <svg class="icon icon-1" viewbox="0 0 256 256">
-                <use xlink:href="/assets/svgs/icons.svg#search"></use>
-              </svg>
-            </button>
-          </span>
-        </div>
-      </div>
-    </form>
+    <a class="full-width btn btn-primary" href="/search">Search Crossroads.net</a>
```

Comment:
Did we not find a way to pass a param into search? Seems like a nice thing to have on the 404 page...I'm not going to stop this from going in due to that, but it seems like passing a param from here into search would be fairly easy with angular?

Reply:
@j-brammer have this story on the backlog to handle: https://rally1.rallydev.com/#/41662702253d/detail/userstory/283954837584